tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CalendarEventParseMethod.scala

```diff
@@ -55,6 +56,9 @@ class CalendarEventMethodModule extends AbstractModule {
     Multibinder.newSetBinder(binder(), classOf[Method])
       .addBinding()
       .to(classOf[CalendarEventRejectMethod])
+    Multibinder.newSetBinder(binder(), classOf[Method])
+      .addBinding()
+      .to(classOf[SupportEmailGetMethod])
```

Comment:
We do not need a method for this.
We just need a capability to be exposed.

Reply:
By including the support email in the returned CapabilityProperties object?

Reply:
Yes!